src/components/Search.js

```diff
@@ -1,15 +1,18 @@
-import React from 'react'
+import React, {Component} from 'react'
 
-const Search = () => {
-  return (
+export default class Search extends Component {
```

Comment:
You don't need to make this a class component since you're just calling the callback prop directly on teh input change.